common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/AbstractRootConcept.scala

```diff
@@ -37,15 +37,22 @@ case class InstantRange(from: Instant,
 object InstantRange {
 
   def apply(from: LocalDate, to: LocalDate): InstantRange =
+    InstantRange(from, to, "")
```

Comment:
I can't really see when we create InstantRange's without a label - where is this?

Reply:
All these tests [here](https://github.com/wellcometrust/catalogue/blob/master/common/internal_model/src/test/scala/uk/ac/wellcome/models/parse/DateParserTest.scala) create an `InstantRange` without a label. The reason for these tests expecting an empty label is that the `DateParser` adds the label as an empty string, and it isn't until [here](https://github.com/wellcometrust/catalogue/blob/master/common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/AbstractRootConcept.scala#L53) that the label is added.

Maybe it would be better if the date parser filled in the label itself (with the label being the whole string being parsed), but I couldn't find a way to do capture the string without the parser consuming the input.

Now you have brought this all up I'm not 100% happy with the way it is, so might take another look..

Reply:
@jamesgorrie have pushed a commit updating this